packages/kit/src/runtime/client/utils.js

```diff
@@ -9,3 +9,13 @@ export function get_base_uri(doc) {
 
 	return baseURI;
 }
+
+/** @param {Window} win */
+export function unselect_all(win) {
+	if (win.getSelection) {
+		const selection = win.getSelection();
+		if (selection) {
+			selection.removeAllRanges();
+		}
+	}
+}
```

Comment:
I believe the global variable `window` is generally avoided in the router code. We could probably simplify this as:

```js
getSelection()?.removeAllRanges()
```

With this one line, maybe we don't need a function after all.